code/modules/uplink/uplink_items.dm

```diff
@@ -1602,6 +1602,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 	cost = 2
 	illegal_tech = FALSE
+	
+/datum/uplink_item/device_tools/rocketboots
+	name = "Rocket Boots"
+	desc = "A pair of heavily modified jump boots with added rocket thrusters. Great for making quick escapes, although you need a very long, unobstructed space to not slam into anything."
+	item = /obj/item/clothing/shoes/bhop/rocket
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+	cost = 3 //you have to choose between this or noslips. Personally I'd go with the noslips.
+	illegal_tech = FALSE //It's a bad idea, not an illegal one.
```

Comment:
why though

Reply:
It's literally a traitor item, this should give you illegal tech.